packages/core/package.json

```diff
@@ -1,8 +1,40 @@
 {
   "name": "@bugsnag/core",
-  "main": "index.js",
+  "main": "src/index.js",
   "version": "8.1.1",
   "types": "types/index.d.ts",
+  "exports": {
```

Comment:
due to the introduction of a `src` folder existing import references such as `@bugsnag/core/lib/es-utils/map` require an export map for them to point to the correct folder (src for now, will migrate to dist is subsequent PRs)

Presumably we'll want to get back to exporting/importing everything from just `@bugsnag/core` in subsequent PRs